src/envoy/transcoding/integration_test.cc

```diff
@@ -0,0 +1,101 @@
+/* Copyright 2017 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "test/integration/integration.h"
+#include "common/grpc/codec.h"
+#include "common/grpc/common.h"
+#include "src/envoy/transcoding/test/bookstore.pb.h"
+
+#include "gtest/gtest.h"
+
+class TranscodingIntegrationTest : public BaseIntegrationTest,
+                                   public testing::Test {
+ public:
+  /**
+   * Global initializer for all integration tests.
+   */
+  void SetUp() override {
+    fake_upstreams_.emplace_back(
+        new FakeUpstream(0, FakeHttpConnection::Type::HTTP2));
+    registerPort("upstream_0",
+                 fake_upstreams_.back()->localAddress()->ip()->port());
+    createTestServer("src/envoy/transcoding/test/integration.json", {"http"});
```

Comment:
This is intended, since all the code in src/envoy/transcoding will be moved to lyft/envoy repo. Taking dependency on our go integration framework doesn't make sense.